resources/views/entities/pages/inventory/_form.blade.php

```diff
@@ -34,7 +34,7 @@
     <div class="col-md-6">
         <div class="form-group required">
             <label>{{ __('entities/inventories.fields.amount') }}</label>
-            {!! Form::text('amount', (empty($inventory) ? 1 : null), ['placeholder' => __('entities/inventories.placeholders.amount'), 'class' => 'form-control', 'maxlength' => 191]) !!}
+            {!! Form::number('amount', (empty($inventory) ? 1 : null), ['class' => 'form-control', 'maxlength' => 191]) !!}
```

Comment:
maxlength doesn't make sense on a numeric, and it's probably worth having a "min" and "max" value based on the database field (hardcoded in the view like for other number fields) :)